setup.py

```diff
@@ -26,7 +26,7 @@ def read(fname):
         "Topic :: Utilities",
         "License :: OSI Approved :: MIT License",
     ],
-    install_requires=["lxml", "requests", "cloudscraper", "garth"],
+    install_requires=["lxml", "requests", "cloudscraper", "garth", "python-dotenv"],
```

Comment:
Please add the dependency in requirements.txt as well.

Reply:
Done. Is there a reason why the dependencies are present in `setup.py` and `requirements.txt`? As developer if you do a `pip install -e .` the dependencies get installed anyway.

Reply:
this is a good question, it startet here https://github.com/jaroslawhartman/withings-sync/commit/8978d1de381e021d95ee2d182262e54d0ad1e99e

Looks like it can be removed, I've added a new Issue https://github.com/jaroslawhartman/withings-sync/issues/133